website/docs/docs/collaborate/git/connect-github.md

```diff
@@ -12,45 +12,65 @@ Connecting your GitHub account to dbt Cloud provides convenience and another lay
 - Clone repos using HTTPS rather than SSH.
 - Trigger [Continuous integration](/docs/deploy/cloud-ci-job) builds when pull requests are opened in GitHub.
 
-
 To connect GitHub in dbt Cloud:
 1. A GitHub organization owner must first [install the dbt Cloud application](/docs/collaborate/git/connect-github#installing-dbt-cloud-in-your-github-account) in your team's GitHub account.
 2. All other dbt Cloud developers on the account need to [personally authenticate with GitHub](/docs/collaborate/git/connect-github#personally-authenticate-with-github) from dbt Cloud.
 
 If you are the GitHub organization owner tasked with the installation of the dbt Cloud app in step 1, you will also be automatically personally authenticated after completion, so step 2 will be taken care of. This means teams of one only need to complete step 1.
 
-:::info Use GitHub On-Premise?
+**GitHub On-Premise**
+
 If you're using an On-Premises GitHub deployment, this method will not work for your account. Please instead reference our docs on [importing a project by git URL](/docs/collaborate/git/import-a-project-by-git-url) to set up your connection. This alternative connection method does not include the benefits of the native integration mentioned above.
-:::
+
+
+## Prerequisites
+
+- You _must_ be a **GitHub org admin** in order to [install the dbt Cloud application](/docs/collaborate/git/connect-github#installing-dbt-cloud-in-your-github-account). 
+- The GitHub org admin _must_ log into dbt Cloud with _Owner_ or _Account Admin_ [permissions](/docs/collaborate/manage-access/self-service-permissions) to integrate a GitHub environment that uses Organizations.
+- You may need to provide an extra dbt Cloud user account for your GitHub org admin until they complete the installation.
```

Comment:
```suggestion
- You may need to temporarily provide an extra dbt Cloud user account with _Owner_ or _Account Admin_ [permissions](/docs/collaborate/manage-access/self-service-permissions) for your GitHub organization owner until they complete the installation.
```

Reply:
yep good callout. i thought of that too and left the self service doc because it included an h2 on enterprise permissions, but i could be more explicit